tests/core/test_axes.py

```diff
@@ -1,182 +1,149 @@
+    simulation_builder.add_person_entity(persons, {'Alicia': {}})
+    simulation_builder.register_variable('salary', persons)
+    simulation_builder.add_parallel_axis({'count': 3, 'name': 'salary', 'min': 0, 'max': 3000, 'period': '2018-11'})
+    simulation_builder.expand_axes()
+    assert simulation_builder.get_input('salary', '2018-11') == approx([0, 1500, 3000])
+    assert simulation_builder.get_count('persons') == 3
+    assert simulation_builder.get_ids('persons') == ['Alicia0', 'Alicia1', 'Alicia2']
+
+
+def test_add_axis_without_period(simulation_builder, persons):
+    simulation_builder.set_default_period('2018-11')
+    simulation_builder.add_person_entity(persons, {'Alicia': {}})
+    simulation_builder.register_variable('salary', persons)
+    simulation_builder.add_parallel_axis({'count': 3, 'name': 'salary', 'min': 0, 'max': 3000})
+    simulation_builder.expand_axes()
+    assert simulation_builder.get_input('salary', '2018-11') == approx([0, 1500, 3000])
+
+
+def test_add_two_axes(simulation_builder, persons):
+    simulation_builder.add_person_entity(persons, {'Alicia': {}})
+    simulation_builder.register_variable('salary', persons)
+    simulation_builder.add_parallel_axis({'count': 3, 'name': 'salary', 'min': 0, 'max': 3000, 'period': '2018-11'})
+    simulation_builder.add_parallel_axis({'count': 3, 'name': 'pension', 'min': 0, 'max': 2000, 'period': '2018-11'})
+    simulation_builder.expand_axes()
+    assert simulation_builder.get_input('salary', '2018-11') == approx([0, 1500, 3000])
+    assert simulation_builder.get_input('pension', '2018-11') == approx([0, 1000, 2000])
+
+
+def test_add_axis_with_group(simulation_builder, persons):
+    simulation_builder.add_person_entity(persons, {'Alicia': {}, 'Javier': {}})
+    simulation_builder.register_variable('salary', persons)
+    simulation_builder.add_parallel_axis({'count': 2, 'name': 'salary', 'min': 0, 'max': 3000, 'period': '2018-11'})
+    simulation_builder.expand_axes()
+    assert simulation_builder.get_count('persons') == 4
+    assert simulation_builder.get_ids('persons') == ['Alicia0', 'Javier1', 'Alicia2', 'Javier3']
+
+
+def test_add_axis_on_group_entity(simulation_builder, persons, group_entity):
+    simulation_builder.add_person_entity(persons, {'Alicia': {}, 'Javier': {}, 'Tom': {}})
+    simulation_builder.add_group_entity('persons', ['Alicia', 'Javier', 'Tom'], group_entity, {
+        'housea': {'parents': ['Alicia', 'Javier']},
+        'houseb': {'parents': ['Tom']},
+        })
+    simulation_builder.register_variable('rent', group_entity)
+    simulation_builder.add_parallel_axis({'count': 2, 'name': 'rent', 'min': 0, 'max': 3000, 'period': '2018-11'})
+    simulation_builder.expand_axes()
+    assert simulation_builder.get_count('households') == 4
+    assert simulation_builder.get_ids('households') == ['housea0', 'houseb1', 'housea2', 'houseb3']
+    assert simulation_builder.get_input('rent', '2018-11') == approx([0, 0, 3000, 0])
```

Comment:
Hey @Morendil , I don't understand why we expect a 0 as the last element of this array.